data/Glitched World/Spirit Temple.json

```diff
@@ -90,7 +90,8 @@
             # access via Early Adult Spirit Temple requires 2 keys (+ jumpslash + explosives)
             # access to Early Adult Spirit Temple guaranteed via can_jumpslash from here
             "Spirit Temple Beyond Central Locked Door": "can_jumpslash and (
-                (Small_Key_Spirit_Temple, 2) or
+                (Small_Key_Spirit_Temple,
+            2) or
```

Comment:
um

Reply:
Sorry, I forgot to turn off a plugin I use for a different project that automatically does layout for me.